### PR TITLE
Bug 1126247 - Hide the Get More Languages link until Marketplace supports langpacks

### PR DESCRIPTION
**Pull request by stasm**
https://bugzilla.mozilla.org/show_bug.cgi?id=1126247

### CODE REVIEW

**Comment by try-server-hook** [comment]
[stasm Staś Małolepszy (stasm)](https://github.com/stasm) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d3254356c9d4)

**Comment by try-server-hook** [comment]
[stasm Staś Małolepszy (stasm)](https://github.com/stasm) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=51bc347bb53d)

File: apps/settings/elements/languages.html [hunk 1]
```diff
@@ -29,7 +29,7 @@ <h1 data-l10n-id="language-header"></h1>
             </select>
           </span>
         </li>
-        <li hidden class="more-languages">
```
Comment (crh0716) [inline apps/settings/elements/languages.html]: Please specify the default value of the key in `common_settings.json`.

**Comment by try-server-hook** [comment]
[stasm Staś Małolepszy (stasm)](https://github.com/stasm) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=dc12e377f4d3)

**Comment by try-server-hook** [comment]
[stasm Staś Małolepszy (stasm)](https://github.com/stasm) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7c1ffd5240ff)

**Pull request merged**